tinygrad/renderer/cstyle.py

```diff
@@ -222,14 +222,32 @@ class OpenCLRenderer(CStyleLanguage):
   float4 = "(float4)"
   code_for_workitem = {"g": lambda x: f"get_group_id({x})", "l": lambda x: f"get_local_id({x})", "i": lambda x: f"get_global_id({x})"}
   uses_vload = True
-  type_map = { dtypes.uint8: "uchar", dtypes.uint32: "uint", dtypes.uint16: "ushort", dtypes.uint64: "ulong" }
+  type_map = { dtypes.uint8: "uchar", dtypes.uint32: "uint", dtypes.uint16: "ushort", dtypes.uint64: "ulong", dtypes.bfloat16: "ushort" }
   def render_cast(self, x, var_dtype, bitcast=False) -> str:
     return f"as_{self.render_dtype(var_dtype)}({x})" if bitcast else super().render_cast(x, var_dtype)
 
   def render_kernel(self, function_name, kernel, bufs, uops, prefix=None) -> str:
-    if any(uop.dtype == dtypes.half for uop in uops): prefix = ["#pragma OPENCL EXTENSION cl_khr_fp16 : enable"]
+    if any(uop.dtype == dtypes.half for uop in uops): prefix = (lambda p: p + prefix if prefix else p)(["#pragma OPENCL EXTENSION cl_khr_fp16 : enable"]) # noqa: E501
```

Comment:
no new noqa E501

Reply:
right now it's at 153 characters, (150 is the limit) I can't think of any other way to do this logic without adding extra lines and while remaining that concise (and not doing code-golfy things like removing spaces). I'll try and think of a way to do it in 150 or less, but if I can't, would you rather have the noqa E501 or an extra line or two?

Reply:
extra line or two is fine, just no more new noqa E501

Reply:
fixed without any new lines